content/dev/code/forms.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Building a Form 
+description: Creating a new form for your shopping cart
```

Comment:
The "your shopping cart" bit feels confusing here.
Perhaps a description more like "Adding a custom form to collect other data on your site" might be more descriptive?